src/applications/personalization/profile-2/msw-mocks.js

```diff
@@ -0,0 +1,62 @@
+import { rest } from 'msw';
```

Comment:
I'm still learning `msw` myself 🙂 Does `setupServer` need to be called, or does `rest.put` do everything that's needed?

Reply:
I actually didn't try as I was basing this off of the docs.